### ISSUE TITLE
I have got a different "deploy net"using create_net with default setting

### ISSUE THREAD

**Issue by my1347** [mgharbi/demosaicnet_caffe#7]
Dear Gharbi
I have got a different "out_put/new_net/deploy.prototxt" comparing with your "pretrained_models/bayer/deploy.prototxt"using "bin/create_net" with default setting.
![image](https://user-images.githubusercontent.com/5409816/35133154-a3ae347a-fd09-11e7-836b-953ac29c61d5.png)
![image](https://user-images.githubusercontent.com/5409816/35133190-c9245edc-fd09-11e7-89ed-3e17ee7a2900.png)

I think this will lead to different result?

**Comment by my1347** [comment]
`

> layer_factory.hpp:69] Check failed: registry.count(type) == 0 (1 vs. 0) Layer type Convolution already registered.`
can not fix it

**Comment by mgharbi** [comment]
the released pre-trained model is based on an older version of the code, accuracy is comparable for both models.
